athena-docdb/src/main/java/com/amazonaws/athena/connectors/docdb/DocDBRecordHandler.java

```diff
@@ -138,26 +141,38 @@ protected void readWithConstraint(BlockSpiller spiller, ReadRecordsRequest recor
             SOURCE_TABLE_PROPERTY, tableNameObj.getTableName());
 
         logger.info("Resolved tableName to: {}", tableName);
+        //Setup Project for Standard Query
+        String disableProjectionAndCasingEnvValue = configOptions.getOrDefault(DISABLE_PROJECTION_AND_CASING_ENV, "false").toLowerCase();
```

Comment:
Why was this moved to here? Can we move it back to keep its declaration close to its usage?

Reply:
done